bin/deploy_www.sh

```diff
@@ -1,13 +1,21 @@
 #!/bin/sh
 
-BUILD="${BUILD:-_build/www}"
+# BUILDWWW corresponds to webroot
+BUILDWWW=~/git/www/visidata.org/_build
```

Comment:
Notice that the previous formulation set only as a default.  Also ~/git/www/... presumes a particular homedir path hierarchy whereas the previous default is relative to the current directory (and by default we run from the cloned dir anyway).

Reply:
I noticed both of those things! Yeah, the real fix was the removal of that `/www` suffix. Everything else may have been an unnecessary modification.

The reason I changed it was to mirror our bin/mkwww. Why does that script hardcode the absolute path and this one work relatively? Perhaps the change here should have been that mkwww should have been a relative path. 

I also removed it as a 'default' because I could not foresee us ever wanting to modify it and it felt like extra boilerplate.